package.json

```diff
@@ -49,5 +49,7 @@
     "eslint": "^7.5",
     "prettier": "^2.4.1",
     "typescript": "~4.4"
-  }
+  },
+  "packageManager": "yarn@1.22.19",
```

Comment:
I know you had to do this for testing, but I'm not sure we want this change in the PR, WDYT?

Reply:
Good catch. Will drop this.